packages/frontend/core/src/components/page-list/docs/page-tags.tsx

```diff
@@ -26,12 +30,17 @@ interface TagItemProps {
 
 export const TagItem = ({
   tag,
+  tagId,
```

Comment:
You only need tag to get the following props. Why do you need to add these props?